src/sentry/search/events/datasets/base.py

```diff
@@ -13,7 +13,7 @@
 class DatasetConfig(abc.ABC):
     custom_threshold_columns: set[str] = set()
     non_nullable_keys: set[str] = set()
-    missing_function_error = InvalidSearchQuery
+    missing_function_error: Any = InvalidSearchQuery
```

Comment:
this is not a fix, this is a hack -- this should probably be `ClassVar[type[Exception]]` I believe?